### PR TITLE
Refactor tests.

### PR DESCRIPTION
**Pull request by siarheishut**
Add using-directive where it is used a lot of times. Remove extra namespaces and constants. Move operators to the same namespace, where the object is declared to GoogleTest is able to use it.

### CODE REVIEW

File: tests/bus_manager_test.cpp [hunk 1]
```diff
@@ -587,20 +589,21 @@ TEST(TestBusManager, TestGetStopInfo) {
...
 
 TEST(TestBusManager, TestFindRouteInfo) {
-  using WaitItem = rm::RouteInfo::WaitItem;
-  using RoadItem = rm::RouteInfo::RoadItem;
-
+  using namespace std;
   using namespace rm;
 
+  using WaitItem = RouteInfo::WaitItem;
+  using RoadItem = RouteInfo::RoadItem;
```
Comment (foxfrikses) [inline tests/bus_manager_test.cpp:596]: rm::

File: tests/bus_manager_test.cpp [hunk 2]
```diff
@@ -587,20 +589,21 @@ TEST(TestBusManager, TestGetStopInfo) {
 }
 
 TEST(TestBusManager, TestFindRouteInfo) {
-  using WaitItem = rm::RouteInfo::WaitItem;
-  using RoadItem = rm::RouteInfo::RoadItem;
-
+  using namespace std;
```
Comment (foxfrikses) [inline tests/bus_manager_test.cpp:592]: To the very top

File: tests/coords_converter_test.cpp [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
 #include "src/coords_converter.h"
 
+using namespace rm::coords_converter;
```
Comment (foxfrikses) [inline tests/coords_converter_test.cpp:9]: minus

File: tests/settings_parser_test.cpp [hunk 4]
```diff
@@ -86,14 +87,14 @@ TEST(TestParseSettings, TestRedneringSettings) {
...
-  auto test_item_without = [&](std::string field) {
+  auto test_item_without = [&](string field) {
     auto item = test_item;
     item.erase(field);
     return item;
   };
-  auto test_item_replace = [&](std::string field, json::Node node) {
+  auto test_item_replace = [&](string field, json::Node node) {
     auto item = test_item;
-    item[field] = std::move(node);
```
Comment (foxfrikses) [inline tests/settings_parser_test.cpp:96]: ?

File: tests/settings_parser_test.cpp [hunk 5]
```diff
@@ -117,159 +118,159 @@ TEST(TestParseSettings, TestRedneringSettings) {
                  MapLayer::kStopLabels}
   };
 
-  auto settings_replace_layers_with = [&](std::vector<MapLayer> layers) {
+  auto settings_replace_layers_with = [&](vector<MapLayer> layers) {
     auto item = settings;
-    item.layers = std::move(layers);
+    item.layers = move(layers);
```
Comment (foxfrikses) [inline tests/settings_parser_test.cpp:123]: ?

**Review by foxfrikses — APPROVED** [review]
LGTM

Please add more info to the commit: what is refactored and why.

File: tests/bus_manager_test.cpp (hunk 1 above)
Comment (siarheishut) [reply tests/bus_manager_test.cpp:596]: Done.

File: tests/bus_manager_test.cpp (hunk 2 above)
Comment (siarheishut) [reply tests/bus_manager_test.cpp:592]: Done.

File: tests/coords_converter_test.cpp (hunk 3 above)
Comment (siarheishut) [reply tests/coords_converter_test.cpp:9]: Done.

File: tests/settings_parser_test.cpp (hunk 4 above)
Comment (siarheishut) [reply tests/settings_parser_test.cpp:96]: Done.

File: tests/settings_parser_test.cpp (hunk 5 above)
Comment (siarheishut) [reply tests/settings_parser_test.cpp:123]: Done.

**Pull request merged**